CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - 2018-07-30
+### Changed
+- Using `"php-school/cli-menu": "^3.0"`
+- Update namespace on CliMenuBuilder
```

Comment:
This changed topic can be only: "Upgraded Cli Menu to v3.".

Reply:
Done.